terraform/032-db-backup/README.md

```diff
@@ -0,0 +1,59 @@
+# 032-db-backup - Database backup service
+This module is used to run mysqldump and backup files to S3
+
+## What this does
+
+ - Create an S3 bucket to store backups
+ - Create a AWS backup user for script to use
+ - Create task definition and ECS service for db-backup
+
+## Required Inputs
+
+ - `app_env` - Application environment
+ - `docker_image` - The docker image to use for this
+ - `ecs_cluster_id` - ID for ECS Cluster
+ - `ecsServiceRole_arn` - ARN for ECS Service Role
+ - `idp_name` - Short name of IdP for use in logs and email alerts
+ - `logentries_set_id` - Logentries logset ID for creating new log in
+ - `mysql_host` - Address for RDS instance
+ - `mysql_pass` - MySQL password for email-service
+ - `mysql_user` - MySQL username for email-service
```

Comment:
Did you mean for this to say `email-service`, or will this need different MySQL credentials that can get to all of the databases that it needs to back up?

Reply:
stupid comments ;-)

Reply:
fixed